api/judicialBooking/data/bookings.mock.data.ts

```diff
@@ -0,0 +1,16 @@
+import { Bookings } from "judicialBooking/models/booking.model";
```

Comment:
Can we call this and the corresponding objects booking as opposed to bookings please?